src/app.ts

```diff
@@ -0,0 +1,69 @@
+import { MikroORM, RequestContext } from '@mikro-orm/core';
+import { PostgreSqlDriver } from '@mikro-orm/postgresql';
+import express, { Application } from 'express';
+import ormConfig from './orm.config';
+import { TrialService } from './components/trial.component/trial.component';
+import { CureWikiService } from './components/curewiki.component/curewiki.component';
+import { NextFunction, Request, Response } from 'express';
+import Container from './utils/helpers/container';
+import { AIService } from './components/ai.component/ai.component';
+import { useExpressServer } from 'routing-controllers';
+import { TrialController } from './controllers/database/trial.controller';
+import 'reflect-metadata';
+
+export class App {
+  host: Application;
+  public orm: MikroORM<PostgreSqlDriver>;
```

Comment:
messy, some explicitly public some implicitly. Cosistency is really important across an entire repo